src/test/java/com/wdtinc/mapbox_vector_tile/adapt/jts/LayerTest.java

```diff
@@ -0,0 +1,113 @@
+package com.wdtinc.mapbox_vector_tile.adapt.jts;
+
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.GeometryFactory;
+import com.vividsolutions.jts.geom.Point;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class LayerTest {
+
+    private static final GeometryFactory GEOMETRY_FACORY = new GeometryFactory();
```

Comment:
Noticed minor typo here, 'T' in factory.

Reply:
np - amended that.

Are you on thespatialcommunity.org's slack channel?  If not it is easy to sign up and we can chat easily about various possible improvements.